### PR TITLE
AEROGEAR-10052: Modify attribute from {mobile-clients}

### PR DESCRIPTION
**Pull request by joaedwar**
## Purpose
**Jira Ref**: https://issues.jboss.org/browse/AEROGEAR-10052

Type:
- [x] Technical change
- [x] Style edit

Check that netlify build displays your changes correctly

PS.: Add images and/or .gifs to illustrate what was changed if this pull request modifies the appearance/output of something presented to the users.

### CODE REVIEW

**Comment by pwright** [comment]
This doesn't work?

Log into the Mobile Developer Console and navigate to the Mobile App in MDC screen.

from https://deploy-preview-814--aerogear-docs-preview.netlify.com/aerogear/latest/getting-started-configuring/

**Comment by pwright** [comment]
another one
pen your Mobile App in MDC in Mobile Developer Console.
from https://deploy-preview-814--aerogear-docs-preview.netlify.com/aerogear/latest/getting-started-running/

**Comment by pwright** [comment]
two in 
https://deploy-preview-814--aerogear-docs-preview.netlify.com/aerogear/latest/identity-management/

The configuration of your Mobile App in MDC in Mobile Developer Console is up-to-date.

Open your Mobile App in MDC in Mobile Developer Console.

File: modules/ROOT/pages/_partials/attributes.adoc [hunk 1]
```diff
@@ -12,7 +12,8 @@
 
 :service-name:
 
-:mobile-client: Mobile App
+:mobile-client: Mobile App in MDC
+:multi-mobile-client: Mobile Apps in MDC
 :mobile-client-openshift: Mobile Client in your OpenShift project
```
Comment (pwright) [inline modules/ROOT/pages/_partials/attributes.adoc]: might need a short version:
```
short-mobile-client: Mobile App

```

**Comment by joaedwar** [comment]
@pwright

**Comment by pwright** [comment]
in getting started:
change from:
your Mobile App in MDC in OpenShift
to
your Mobile App in MDC

**Comment by pwright** [comment]
https://deploy-preview-814--mobile-docs.netlify.com/aerogear/latest/mobile-developer-console/

The Mobile Services tab in the Mobile App in MDC view shows the provisioned services that are available to be bound to the Mobile App in MDC. If a service has not yet been bound to a Mobile App in MDC, a binding can be created by selecting the Bind to App button.

to 

The Mobile Services tab in the Mobile App view shows the provisioned services that are available to be bound to the Mobile App. If a service has not yet been bound to a Mobile App in MDC, a binding can be created by selecting the Bind to App button.

**Comment by pwright** [comment]
same url, "The Builds tab in the Mobile App in MDC view"
maybe search for view, that should use the short 'Mobile App' attribute

**Comment by pwright** [comment]
same url:
In order to build the Mobile App in MDC app
In order to build the Mobile App

**Comment by joaedwar** [comment]
@pwright changes made

**Comment by pwright** [comment]
only a few left ;)

https://deploy-preview-814--aerogear-docs-preview.netlify.com/aerogear/latest/ds-openshift/

Click on the Mobile App in MDC on the Overview screen

https://deploy-preview-814--aerogear-docs-preview.netlify.com/aerogear/latest/security/

Click on the Mobile App in MDC on the Overview screen

**Pull request merged**